meteor/client/ui/Shelf/DashboardPanel.tsx

```diff
@@ -344,7 +344,6 @@ export class DashboardPanelInner extends MeteorReactComponent<
 		if (queue && sourceLayer && !sourceLayer.isQueueable) {
 			console.log(`Item "${adlibPiece._id}" is on sourceLayer "${adlibPiece.sourceLayerId}" that is not queueable.`)
 			return
-			// TODOSYNC: TV2 uses queue = false
```

Comment:
Is there a reason for the TODOSYNC being removed, without any changes?

Reply:
Yes. After talking with Jacob the behaviour that setting queue to false was something that we didn't want.